### PR TITLE
High: aws-vpc-route53: Add agent for AWS Route 53 (fate#322781)

### PR DESCRIPTION
**Pull request by krig**
This agent changes a private zone Route 53 agent to
the appropriate IP address. This agent has been
developed in collaboration of SUSE and Amazon/AWS.

### CODE REVIEW

File: heartbeat/aws-vpc-route53 [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+	# profile
+	[[ -z "$OCF_RESKEY_profile" ]] && ocf_log error "AWS CLI profile not set $OCF_RESKEY_profile!" && exit $OCF_ERR_CONFIGURED
+
+	# TTL
+	[[ -z "$OCF_RESKEY_ttl" ]] && ocf_log error "TTL not set $OCF_RESKEY_ttl!" && exit $OCF_ERR_CONFIGURED
+
+	ocf_log debug "Testing aws command"
+	aws --version 2>&1
+	if [ "$?" -gt 0 ]; then
+		error "Error while executing aws command as user root! Please check if AWS CLI tools (Python flavor) are properly installed and configured." && exit $OCF_ERR_INSTALLED
```
Comment (oalbrigt) [inline heartbeat/aws-vpc-route53]: "ocf_log" is missing here.

Comment (krig) [inline heartbeat/aws-vpc-route53]: nice catch!

**Comment by oalbrigt** [comment]
Looks good to me.

**Comment by krig** [comment]
Thank you for reviewing! Merged.

**Pull request merged**